iotdb-core/datanode/src/main/java/org/apache/iotdb/db/schemaengine/schemaregion/mtree/impl/pbtree/schemafile/AliasIndexPage.java

```diff
@@ -274,21 +278,20 @@ public String splitByKey(String key, String entry, ByteBuffer dstBuffer, boolean
 
     int sp; // virtual index to split
     if (monotonic) {
-      // new entry into part with more space
-      sp = key.compareTo(lastKey) > 0 ? Math.max(pos, n / 2) : Math.min(pos, n / 2);
+      // New entry into part with more space
+      sp = key.compareTo(lastKey) > 0 ? Math.max(pos, n >> 1) : Math.min(pos, n >> 1);
     } else {
-      sp = n / 2;
+      sp = n >> 1;
     }
 
-    // little different from InternalSegment, only the front edge key can not split
+    // Little different from InternalSegment, only the front edge key can not split
     sp = sp <= 0 ? 1 : sp;
 
-    // this method BREAKS envelop of the passing in buffer to be more efficient
+    // This method BREAKS envelop of the passing in buffer to be more efficient
     // attributes for dstBuffer
-    short memberNum = 0,
-        spareOffset = (short) dstBuffer.capacity(),
-        spareSize = (short) (spareOffset - SchemaFileConfig.PAGE_HEADER_SIZE);
-    long nextSegAddress = this.nextPage;
```

Comment:
long nextSegAddress = this.nextPage; is absent?

Reply:
This variable is not used at all...